com.github.marinm.songrec.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "com.github.marinm.songrec",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "songrec",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host:ro",
+        "--filesystem=/tmp:ro",
+        "--filesystem=xdg-data",
```

Comment:
This shouldn't be technically needed. Your application should follow `$XDG_DATA_HOME` which flatpak automatically grants access to.

Reply:
Thanks, fixed in commit c879611